src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENTAL_DAY_RATE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return ((days * RENTAL_DAY_RATE) - LONG_TERM_DISCOUNT);
+  }
+
+  if (days < SHORT_TERM) {
+    return (days * RENTAL_DAY_RATE);
+  }
+
+  return ((days * RENTAL_DAY_RATE) - SHORT_TERM_DISCOUNT);
```

Comment:
```suggestion
  return days * RENTAL_DAY_RATE - SHORT_TERM_DISCOUNT;
```
parentheses redundant in all cases - just FYI